events_backend/app/views.py

```diff
@@ -156,7 +156,7 @@ def post(self, request):
 
 class OrgDetail(APIView):
     #TODO: alter classes to token and admin?
-    authentication_classes = (TokenAuthentication, )
+    authentication_classes = ()#(TokenAuthentication, )
     permission_classes = (permissions.IsAuthenticated, )
```

Comment:
I'm assuming this isn't commented out because we need to verify the organizations have an org email in the db?